content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,28 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+### Pod Deletion Cost 
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Using the `controller.kubernetes.io/pod-deletion-cost` annotation, users can set a preference regarding 
+which pods to remove first when downscaling a ReplicSet.
+
+The annotation should be set on the pod, and the expected type is `int32`. It represents the cost of
+deleting a pod compared to other pods belonging to the same ReplicaSet. Pods with lower deletion
```

Comment:
nit: How do you determine the cost of deletion? How do you know what number to set (are there other factors)?